schemas/authenticate-hawk-request.yml

```diff
@@ -43,9 +43,9 @@ properties:
       **Note,** order of querystring elements is important.
   host:
     type:                 string
```

Comment:
Sorry, I got it wrong... I remove the other `type: string`, I don't know if this works... But why wouldn't it, I suspect it does :)